lib/ceremony.hoon

```diff
@@ -243,7 +253,9 @@
             =<  net
             %+  fall
               (~(get by deed-map) par)
-            (~(got by tlon-map) par)
+            %+  fall
+              (~(get by tlon-map) par)
+            *rights
```

Comment:
iirc if it's not in `deed-map` it *needs* to be in `tlon-map`. I'm guessing you hit the Tlon stars with this?

Reply:
`fall` is not a macro, so this failed if the ship was not in `tlon-map` even if it *was* in `deed-map` (because the `got:by` got called anyway).

Reply:
oh christ, I feel so dumb for getting stuck on this some time ago lol